ui/page/collection/internal/collectionActions/internal/playButton/index.js

```diff
@@ -0,0 +1,28 @@
+import { connect } from 'react-redux';
+import { selectClaimForUri } from 'redux/selectors/claims';
+import { selectUrlsForCollectionId } from 'redux/selectors/collections';
+import ShuffleButton from './view';
```

Comment:
There are a bunch of `ShuffleButton` from copy-pasting.  Rename these for clarity.